package.json

```diff
@@ -56,7 +56,7 @@
     "js-utils": "github:jitsi/js-utils#964294793a7dbb85b2a38c87abe5ae46c4acf0af",
     "jsrsasign": "8.0.12",
     "jwt-decode": "2.2.0",
-    "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#4e7034ee6a0d0e68487c583c981f4e07ab73926c",
+    "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#d5d8e0a1e4c6200cc0dd73963b7c4786326d45ec",
```

Comment:
Is the lib change intentional here?

Reply:
Yes, the lib change is for taking the changes that were made in LJM for fixing the presenter mode on Firefox.

Reply:
Okay. I think it would be best to keep the commits separate, but that may be more personal preference and not necessary to split now.

Reply:
Ok, will keep the commits separate from now on.